rotkehlchen/assets/converters.py

```diff
@@ -890,3 +891,16 @@ def asset_from_iconomi(symbol: str) -> Asset:
         raise UnsupportedAsset(symbol)
     name = ICONOMI_TO_WORLD.get(symbol, symbol)
     return symbol_to_asset_or_token(name)
+
+
+def asset_from_uphold(symbol: str) -> Asset:
```

Comment:
Can you add a test for this just like we have for all other exchanges? Similar to https://github.com/rotki/rotki/blob/b9660102272522cbba7e438c05500149e64dfe98/rotkehlchen/tests/exchanges/test_gemini.py#L60-L79

Just a test that would go through all assets from the mapping you introduced and make sure they are all recognized as assets (or are a known exception)